src/og/quadTree/Node.js

```diff
@@ -249,7 +249,7 @@ class Node {
 
             if (seg.tileZoom < 2 && seg.normalMapReady) {
                 this.traverseNodes(cam, maxZoom, terrainReadySegment, stopLoading);
-            } else if (seg.terrainReady && (!maxZoom && cam.projectedSize(seg.bsphere.center, seg._plainRadius) < planet._lodSize || maxZoom && ((seg.tileZoom === maxZoom) || !altVis))) {
+            } else if (seg.terrainReady && (!maxZoom && cam.projectedSize(seg.bsphere.center, seg._plainRadius) < planet._lodSize / Math.pow(2, planet._zoomOffset) || maxZoom && ((seg.tileZoom === maxZoom) || !altVis))) {
```

Comment:
Not sure it's the correct decision. Because, it just affects the level of detail, but in the offset case segment must be rendered with the offset, any segment. Need to think.

Reply:
Hey @program-sam! Sorry, for the long responses! Could you please update your PR, and add some examples in the sandbox (folder), I could check it,  please! Cheers!

Reply:
Hi @Zemledelec , I just added a commit with an example added in the sandbox folder. I do agree there might be better ways so this applies to one layer only (playing with the node material code i suppose). This, as you mention, changes the global level of detail indeed, (which might also be desirable?)

Reply:
Looks like zoomOffset works for all nodes! I think that it must work for selected layers.

Reply:
...it's harder to implement, and it's what I'm talking about. But, I see that it affects to _lodSize parameter, and actually it could be optimized in the _lodSize))...but...we can make the change if it's ok for you, but need to optimize it in _lodSize already includes. No need for math in the if.

Reply:
Totally agreed, the ideal zoomOffset is not for all nodes, but just for a specific layer. And I see how that is a lot harder to implement. Maybe instead of calling this suggested code a zoomOffset, it could be simply used to make the lodSize accessible? And then _lodSize is only changed on initialization. But I do not know of lodSize impacts different parts of openglobus

Reply:
For example, try in console, `__globus__.planet.setLodSize(120,120)` where 120 is a projected (approximately) size of a node on the screen in pixels. It's the maximal node size we stop to traverse the quadtree in the depth. It means that you will see all nodes 120px (approximately) size. If you set 512, each node will be 512px in the screen. The less value the more nodes on the viewport, and the better quality. Second parameter (also 120) is the size of distant nodes when the camera looks at the horizon.

Unfortunately, I can't promise when I will make the zoomOffset feature, but it is definitely a nice and useful feature...

Reply:
oh ok! I didn't realize the setLodSize was already a possibility. This already works almost exactly as what it is that this suggested code does. Do nodes have a fixed texture resolution? You mention a lower LOD to be a higher quality (so I assume a lower LOD to be less quality). But I am making canvas tiles in the same resolution as the LOD size. Am I correct to assume that these larger canvasses compensate for the larger LOD?

Reply:
> Do nodes have a fixed texture resolution?
Basically Node(aka Segment, tile)contains a mesh (segment), you can use any texture size and resolution, it depends on the image size, for instance, CanvasTiles can handle any texture size for a segment.

> You mention a lower LOD to be a higher quality (so I assume a lower LOD to be less quality)
I mean that LOD in terms of a node is the Node zoom level. The closer to the surface, the more detailed the segment mesh could be. It doesn't mean that you have to use 512x512px image, but it means that the image (or terrain mesh) contains a higher resolution such as meters in pixels.

> But I am making canvas tiles in the same resolution as the LOD size. Am I correct to assume that these larger canvasses compensate for the larger LOD?
The first sentence is ok:) The second is "no" because image resolution doesn't affect the projected size of a segment(node) to the screen, only mesh size 3D affects the projected size. It means that if you use 512x512 image size instead of 256x256 on default (LOD 230 value) the image will be more detailed, not the mesh, but the image only.

Hope it's clear, if not I'll try to depict something. ZoomOffset more interesting feature than ordinary LOD, because it makes mesh detail and image resolution work separately and customizable.

Reply:
Ok thanks! more than clear and this is how I thought it worked. Thanks for clarifying further! My apologies for the possible confusion by using the words quality, resolution, detail etc. vaguely and interchangeably.

For me personally the .setLodSize function on planet is sufficient for my use and this PR can be closed, until there is the time, or more use cases for a layer-specific zoomOffset 👍